### ISSUE TITLE
Feature: allow users to specify kernel modules directory when running with custom kernel

### ISSUE THREAD

**Issue by brb** [cilium/little-vm-helper#117]
At the time of writing, the LVH runner allows users to use a custom kernel path (to bzImage) via `--kernel` cmdline param. Unfortunately, it doesn't allow users to specify kernel modules directory. The pre-installed kmods in a VM image usually cannot be used with the custom kernel.

One workaround is to mount linux dir via `--host-mount`, and then from the VM `cd /host/linux && make module_install`. However, having the explicit kmods param would make the LVH runner easier to use.

**Comment by SAMurai-16** [comment]
@brb is this issue still up for grabs??

**Comment by brb** [comment]
Yep, it has not been implemented yet. Before taking a stab, mind elaborating your approach?

**Comment by SAMurai-16** [comment]
Current LVH allows booting a custom kernel with `--kernel`, but the VM continues to use whatever `/lib/modules` directory is already present in the image. When the custom kernel version differs from the image’s preinstalled modules, module loading fails.

The approach I’m considering is to add a `--kernel-modules` runner option so users can provide the modules directory that matches the custom kernel. LVH would then make that directory available to the guest at runtime, avoiding the need to rebuild the VM image or manually mount the full Linux source tree and run make `module_install` inside the VM.
